src/common/src/block_builder.cc

```diff
@@ -0,0 +1,147 @@
+#include "block_builder.h"
+#include <bits/stdc++.h>
+
+BlockBuilder::BlockBuilder(void)
+{
+}
+
+BlockBuilder::BlockBuilder(const std::vector<std::vector<std::string> > &records)
+{
+    for (size_t i = 1; i < records.size(); ++i)
+    {
+        const auto&  record = records[i];
+        types::Block block  = ConvertRecordToBlock(record);
+        blocks_.push_back(block);
+    }
+}
+
+void AssignBlockInfrastructure(types::Block &block, const std::string &infrastructure)
+{
+    std::string infra = infrastructure;
```

Comment:
Let's rename these so we don't have two variables named the same thing, where one is an abbreviation of the other.

Reply:
changed infrastructure to input_string